server/chess.py

```diff
@@ -62,13 +62,14 @@ def path_is_empty(
         """
         file_delta = file - piece.file
         rank_delta = rank - piece.rank
-        assert abs(file_delta) == abs(rank_delta), 'Path must be straight.'
+        """Commented out as only does diagonals, can add orthogonals as well if you want
+        """assert abs(file_delta) == abs(rank_delta), 'Path must be straight.'
```

Comment:
Good catch, though comments should be inserted by starting the line with the \# character. And in general, there is no need to commit commented out lines, they can just be removed.